### PR TITLE
serials media endpoint was implemented

### PR DESCRIPTION
**Pull request by KravaAO**
I've added an endpoint for series. If everything looks good, I plan to proceed with adding additional endpoints. Once all endpoints are successfully added, I'll work on removing any code duplication.

### CODE REVIEW

File: src/parse/parse.service.ts [hunk 1]
```diff
@@ -14,6 +14,15 @@ export class ParseService {
         }
     }
 
+    async fetchSerials(url = 'https://uaserial.club/serial/'): Promise<string> {
```
Comment (barbinkh) [inline src/parse/parse.service.ts]: we can use fetchContent. Don't need to create one more function

File: src/parse/parse.service.ts [hunk 2]
```diff
@@ -71,4 +80,14 @@ export class ParseService {
             countOfPages: countOfPages,
         }
     }
+
+    async fetchSerial(url: string): Promise<MediaResponse> {
```
Comment (barbinkh) [inline src/parse/parse.service.ts]: we can use fetchMedia. Don't need to create one more function

File: src/parse/parse.controller.ts [hunk 3]
```diff
@@ -17,4 +17,9 @@ export class ParseController {
     async fetchAll(@Param('page') page = '1'): Promise<MediaResponse> {
         return await this.parseService.fetchMedia(`https://uaserial.club/${page}`)
     }
+
+    @Get('/serials/:page')
+    async fetchSerials(@Param('page') page = '1'): Promise<MediaResponse> {
+        return await this.parseService.fetchSerial(`https://uaserial.club/serial/${page}`);
```
Comment (barbinkh) [inline src/parse/parse.controller.ts]: Please use fetchMedia functions for parsing series

File: src/parse/parse.controller.ts [hunk 4]
```diff
@@ -17,4 +17,9 @@ export class ParseController {
     async fetchAll(@Param('page') page = '1'): Promise<MediaResponse> {
         return await this.parseService.fetchMedia(`https://uaserial.club/${page}`)
     }
+
+    @Get('/serials/:page')
```
Comment (barbinkh) [inline src/parse/parse.controller.ts]: /series/:page

**Review by barbinkh — CHANGES_REQUESTED** [review]

File: src/parse/parse.controller.ts (hunk 4 above)
Comment (KravaAO) [reply src/parse/parse.controller.ts]: +

File: src/parse/parse.controller.ts (hunk 3 above)
Comment (KravaAO) [reply src/parse/parse.controller.ts]: +

File: src/parse/parse.service.ts (hunk 2 above)
Comment (KravaAO) [reply src/parse/parse.service.ts]: +

File: src/parse/parse.service.ts (hunk 1 above)
Comment (KravaAO) [reply src/parse/parse.service.ts]: +

**Review by barbinkh — APPROVED** [review]

**Pull request merged**